start-from-zero/index.css

```diff
@@ -246,21 +350,208 @@ textarea {
 
 #select {
     float: right;
+    -webkit-appearance: none; 
+    -moz-appearance: none;
+    appearance: none;   
+    background-image: url(assets/down-arrow.png);
+    background-repeat: no-repeat;
+    background-position: 90% 50%;
+    background-size: 13px;
+    position: relative;
+    height: 20px;
+    
+}
+
+
+
+textarea {
+    width: 99.7%;
+    margin: 0 auto;
+    padding: 0;
+}
+
+
+#footer-new {
+    position: relative;
 }
 
 fieldset {
+    margin: 30px 0;
+}
+
+.fieldset-wrap {
+    padding: 10px 10px;
+}
+
+fieldset legend {
+    padding: 0 20px;
+}
+
+#inner-li {
+    height: 100%;
+}
+
+.ultimate-wrap {
     width: 100%;
+    margin: 0 auto;
 }
 
-#footer-new {
+.outer-ul, .down-ul {
+    height: 100%;
+    display: inline-flex;
+    flex-direction: row;
+    justify-content: space-between;
+}
+
+.down-ul {
     position: relative;
-    
+    bottom: 80px;
+}
+
+#height-100 {
+    height: 100%;
+}
+
+.inner-ul li {
+    height: 100%;
+    line-height: 80px;
+    position: relative;
+    z-index: 1000;
+}
+
+
+
+.position-div {
+    display: inline-flex;
+    flex-direction: row;
+    align-items: center;
+}
+
+.position-div div {
+    background: white;
+    padding: 0 30px;
+    transition: all 300ms ease;
+}
+
+.position-div div a {
+    text-decoration: none;
+    color: black;
+}
+
+.one,.two {
+    height: 73px;
+    line-height: 73px;
+    position: relative;
+    z-index: 10000;
+}
+
+
+.position-div div:hover {
+    box-shadow: 0px 5px 0px 0px #03A9F4;
+}
+
+
+.tr-first td {
+    width: 25%;
+    text-align: center;
+}
+
+.tr-second td {
+    width: 25%;
+    text-align: center;
+}
+
+
+
+.tr-first {
+    height: 40px;
+    color: white;
+    background: #03A9F4;
+    padding-top: 10px;
+}
+
+.tr-third {
+    width: 25%;
+    text-align: center;
+}
+
+.tr-third td {
+    border-bottom: 1px solid black;
+    border-top: 1px solid black;
+}
+
+.tr-third td:last-child{
+    border-right: 1px solid black;
+}
+
+.tr-third td:first-child{
+    border-left: 1px solid black;
+}
+
+.fieldset-wrap .row1 {
+    padding: 7px 0;
+}
+
+h4{
+    color: #03A9F4;
+    padding-left: 40px;
+    margin-bottom: 7px;
+}
+
+select {
+    width: 40%;
+}
+
+.button-send {
+    width: 9%;
+    background: #03A9F4;
+    align-self: flex-end
+}
+
+.description {
+    font-size: 13px;
+}
+
+option,select {
+    font-size: 13px;
+}
+
+td div {
+    display: inline-block;
+    height: 24px;
+    width: 24px;
+    background: #03A9F4;
+    text-align: center;
+    position: relative;
+    top: 2px;
+    color: white;
+    font-size: 24px;
+    border-radius: 100px;
+}
+
+td div span {
+    position: relative;
+    bottom: 2px;
     
 }
 
+td div span a {
```

Comment:
I dunno, this kinda feels like it's too specific. A common best practice is to limit the amount of nesting in your CSS selectors to at most 3 levels. There's usually a better way of doing things than just chaining a bunch of tag names together, that have the advantage of not breaking as soon as your markup changes. Dare I say it? CSS classes FTW!